### ISSUE TITLE
The inner surfaces of all Sherman and M3 halftrack treads move backwards, opposite to the direction of the outer treads

### ISSUE THREAD

**Issue by handsm** [DarklightGames/DarkestHour#688]
The texture mapping of the treads is back to front and needs re-doing. As the treads rotate as the tank moves, the inner faces of the treads can be seen to move in the direction, the opposite of how the outer faces are moving, which looks pretty bizarre. Affects all Sherman variants, even though they use different hull meshes and tread textures.

┆Issue is synchronized with this [Trello card](https://trello.com/c/EQASOTlK)

**Comment by cmbasnett** [comment]
Makes you wonder what the previous devs were smoking!

**Comment by AndrewTheel** [comment]
They looked at the player faces and were like... meh my treads are good enough.

**Comment by handsm** [comment]
It's an odd one. Although it should be really obvious, I never noticed it for years, and I've spent a lot of time working on and looking at vehicles. I think maybe the eye focuses on the outside of the treads and the inner faces become a kind of blind spot. But having become aware of the glitch, now when I look at moving Sherman treads my eyes go haywire!

**Comment by handsm** [comment]
Firefly treads fixed as part of Firefly model re-work in commit https://github.com/DarklightGames/DarkestHour/commit/3bfa8a42cbf2674e22c90003b7669c8543a86b94.

**Comment by handsm** [comment]
Fixed in commit https://github.com/DarklightGames/DarkestHour/commit/b1340090be7ccd59a98c15dc9a6c6789011c9803.

As well as re-mapped treads for the Sherman M4A1 and M4A3 models, I've tweaked the Firefly treads  fixed in the last commit (destroyed vehicle static mesh and damaged tracl static meshes updated to match).

Included damaged/broken left and right tread static meshes for all Sherman variants, which are shown if tank's treads get damaged.
